### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by OleksiiSym**
[DEMO LINK](https://oleksiisym.github.io/layout_landing-page/)

### CODE REVIEW

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
1. don't see your solution
image

**Comment by OleksiiSym** [comment]
Sorry for that, edited already

**Comment by OleksiiSym** [comment]
> 

Sorry for that, edited already

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good start 👍 
Let's to fix some moments
Call number don't working. Also add a phone number when on hover to call
image
Also add transition when this menu appears at page
image

**Comment by OleksiiSym** [comment]
> Good start 👍 Let's to fix some moments Call number don't working. Also add a phone number when on hover to call image Also add transition when this menu appears at page image

Thanks, fixed issue with phone icon but dont know how to make transition for menu appearance. May you give some advice?

**Comment by vlad-demchuk** [comment]
if you use `transfom` for menu, just add `transition: transform 0.3s` for the element  for example
feel free to ask your question it in fe_chat

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Good work!
Just add hover effects to all interactive elements
image
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Let's improve your page
1. Add favicon to page
image

2. Need to show phone number when hovering this icon
image

3. Need to remove scroll when this menu is open
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Add placeholder text for all input
image

2. Add hover effects for all clickable elements
image

3. Add `cursor: pointer` and transition for this button
image

4. Need to make all these fields are required
image

5. I get this error after send form
image

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,257 @@
...
+
+        <div class="about__content">
+          <div class="conteiner about__content--grid">
+            <h2 class="section-title section-title--block">Timeless, for 50 years and counting</h2>
+            <p class="about__description">Bang & Olufsen is expanding its Recreated Classics Program with the launch of Beosystems – a limited edition music system that transcends time by bridging the gap between one of our iconic designs from 1972 and today’s cutting-edge digital technology.</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="contact conteiner">
```
Comment (TarasHoliuk) [inline src/index.html:198]:
```suggestion
      <section class="contact container">
```
typo

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,257 @@
...
+        <h2 class="section-title section-title--inline">Contact us</h2>
+
+        <form
```
Comment (TarasHoliuk) [inline src/index.html:201]: have you tried it (the [checklist](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md))?
image

**Comment by TarasHoliuk** [comment]
Checklist 6:
image

Checklist 7:
image

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
GJ :+1: 
Almost done - carefully read the checklist and ensure you achieved all goals

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done! Consider fixing validation errors (mostly it's mistyping of tag names): 
Screenshot 2023-08-06 at 15 25 53
Screenshot 2023-08-06 at 15 26 04
Screenshot 2023-08-06 at 15 27 36
Screenshot 2023-08-06 at 15 23 41
Check margins between products.
Screenshot 2023-08-06 at 15 21 44
These links should be anchor links. Consider also adding transitions for hovers

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Need to change autocomplete styles for input
image

2. Add transition for hover effects here
image

**Review by anastasiiavorobiova — APPROVED** [review]
Looks outstanding!

**Pull request closed without merging**